packages/components/src/components/form/BottomText.tsx

```diff
@@ -10,14 +10,14 @@ import { InputState } from './inputTypes';
 export const BOTTOM_TEXT_MIN_HEIGHT = 26; // 1 line of text + top padding
 
 const slideDown = keyframes`
-    from {
-        opacity: 0;
-        transform: translateY(-2px);
-    }
-    to {
-        opacity: 1;
-        transform: translateY(0);
-    }
+from {
```

Comment:
why this change (blank space only?)?

Reply:
I think it was a linter conflict, removed to the previous state in here https://github.com/trezor/trezor-suite/pull/13623/commits/5e2a396fd96765f4330a71cf21ce68b46196a2f9